docs/README_id.md

```diff
@@ -0,0 +1,410 @@
+<h1 align="">Data Wilayah Indonesia (<i>Indonesian Area Data</i>)</h1>
+
+[![npm version](https://img.shields.io/npm/v/idn-area-data.svg)](https://www.npmjs.com/package/idn-area-data)
+[![npm downloads](https://img.shields.io/npm/dm/idn-area-data.svg)](https://www.npmjs.com/package/idn-area-data)
+[![GitHub license](https://img.shields.io/github/license/fityannugroho/idn-area-data.svg)](LICENSE)
+
+Menyediakan data wilayah administratif di Indonesia, mulai dari tingkat provinsi, kabupaten, kecamatan, hingga desa berdasarkan sumber resmi terkini. Modul ini juga menyediakan data pulau sejak versi 1.3.0, dan **data provinsi PAPUA baru** sejak versi 2.0.0.
```

Comment:
Please **change all translation** for some of the words listed below:

| English Word | Translation |
|--------|--------|
| "regency" and "regencies" | "kabupaten/kota" |
| "village" and "villages" | "desa/kelurahan" |

Adjust the capitalization as well, for example "Regency" into "Kabupaten/Kota".

```suggestion
Menyediakan data wilayah administratif di Indonesia, mulai dari tingkat provinsi, kabupaten/kota, kecamatan, hingga desa/kelurahan berdasarkan sumber resmi terkini. Package ini juga menyediakan data pulau sejak versi 1.3.0, dan **data provinsi PAPUA baru** sejak versi 2.0.0.
```

Reply:
@depimomo are you still working on this? Let me know if there are any difficulties